definitions/types/cidr.json

```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema",
   "type": "string",
+  "pattern": "^([0-9]{1,3}\.){3}[0-9]{1,3}(\/([0-9]|[1-2][0-9]|3[0-2]))?$"
```

Comment:
I think Json schema has a message field too. We should use that in conjunction with the regex to provide a useful error message to the end user.